src/agent/onefuzz-telemetry/src/lib.rs

```diff
@@ -66,7 +66,7 @@ impl fmt::Display for InstanceTelemetryKey {
     }
 }
 
-pub type TelemetryClient = appinsights::TelemetryClient<appinsights::InMemoryChannel>;
+pub type TelemetryClient = appinsights::TelemetryClient;
```

Comment:
Note that the channel is no longer explicit in the type. However, the default is the `InMemoryChannel`.